be/src/vec/common/hash_table/hash_table.h

```diff
@@ -245,11 +245,14 @@ struct HashTableGrower {
     /// The state of this structure is enough to get the buffer size of the hash table.
     doris::vectorized::UInt8 size_degree = initial_size_degree;
 
+    size_t _buf_size = 1ULL << initial_size_degree;
+    size_t _max_fill = 1ULL << (initial_size_degree - 1);
```

Comment:
I think we'd better not change this. I find that it only improved about 0.02%. But the readability of the code is affected. Maybe @HappenLee  could read this PR.